Index: administrator/language/en-GB/en-GB.com_config.ini
===================================================================
--- administrator/language/en-GB/en-GB.com_config.ini	(revision 10720)
+++ administrator/language/en-GB/en-GB.com_config.ini	(working copy)
@@ -159,7 +159,7 @@
 TIPFTPUSERNAME=The username used to access the FTP server.
 TIPGLOBALSITEMETATADESC=Enter a description of the overall Web site that is to be used by search engines. Generally, a maximum of 20 words is optimal.
 TIPGLOBALSITEMETAKEY=Enter the keywords and phrases that best describe your Web site. Separate keywords with a comma and group phrases within single quotations.
-TIPHELPSERVER=Select the name of the Help Server where your system will collect the Help Screen displays from. Set to Local to use those files within the core installation of Joomla!. Click the Refresh button to update the Help Server List.
+TIPHELPSERVER=Select the name of the Help Server from which your system will collect the Help Screen displays. Set to Local to use those files within the core installation of Joomla!. Click the Reset button to restore the default help server. Warning: Pressing Reset will erase any custom help servers you have added.
 TIPIFSETTOSHOWAUTHOR=If set to Show, the Name of the Author will be displayed.  This a global setting but can be changed at Menu and Article levels.
 TIPIFSETTOSHOWDATETIMECREATED=If set to Show, the date and time an Article was created will be displayed. This a global setting but can be changed at Menu and Article levels.
 TIPIFSETTOSHOWDATETIMEMODIFIED=If set to Show, the date and time an Article was last modified will be displayed. This a global setting but can be changed at Menu and Article levels.
Index: administrator/components/com_config/views/application/tmpl/config_system.php
===================================================================
--- administrator/components/com_config/views/application/tmpl/config_system.php	(revision 10720)
+++ administrator/components/com_config/views/application/tmpl/config_system.php	(working copy)
@@ -41,7 +41,7 @@
 			</td>
 			<td>
 				<?php echo $lists['helpsites']; ?>
-				<input type="button" onclick="submitbutton('refreshhelp')" value="<?php echo JText::_('Refresh'); ?>" />
+				<input type="button" onclick="submitbutton('refreshhelp')" value="<?php echo JText::_('Reset'); ?>" />
 			</td>
 		</tr>
 		</tbody>
