Index: administrator/components/com_cache/toolbar.cache.html.php
===================================================================
--- administrator/components/com_cache/toolbar.cache.html.php	(revisão 11329)
+++ administrator/components/com_cache/toolbar.cache.html.php	(cópia de trabalho)
@@ -26,7 +26,7 @@
 	*/
 	function _DEFAULT() {
 
-		JToolBarHelper::title( JText::_( 'Cache Manager' ), 'cache.png' );
+		JToolBarHelper::title( JText::_( 'Cache Manager' ), 'checkin.png' );
 		JToolBarHelper::custom( 'delete', 'delete.png', 'delete_f2.png', 'Delete', true );
 		JToolBarHelper::help( 'screen.cache' );
 	}
