Index: components/com_content/models/category.php
===================================================================
--- components/com_content/models/category.php	(revision 11384)
+++ components/com_content/models/category.php	(working copy)
@@ -487,6 +487,7 @@
 			{
 				// clean filter variable
 				$filter = JString::strtolower($filter);
+				$hitsFilter = intval($filter);
 				$filter	= $this->_db->Quote( '%'.$this->_db->getEscaped( $filter, true ).'%', false );
 
 				switch ($params->get('filter_type'))
@@ -500,7 +501,7 @@
 						break;
 
 					case 'hits' :
-						$where .= ' AND a.hits LIKE '.$filter;
+						$where .= ' AND a.hits >= '.$hitsFilter. ' ';
 						break;
 				}
 			}
